scripts/visualize/visualize.matthew_map.R

```diff
@@ -1,11 +1,13 @@
 visualize.matthew_map <- function(viz){
   
-  counties <- readData(viz[['depends']])
+  counties <- readData(viz[['depends']][1])
```

Comment:
The depends section is really just to "build" the dependency graph, you should add a field where dependencies are actually told what to do.